frontend/src/deviceDetailsPage/DeviceInfoTable/DeviceInfoRow.jsx

```diff
@@ -2,15 +2,23 @@ import React from "react";
 import Proptypes from "prop-types";
 
 import { TableContainer, TableCell, TableRow } from "@material-ui/core";
-import ChannelDetailsTable from "../devicelist/ChannelDetailsTable";
-import InputChannelInfo from "../model/InputChannelInfo";
-import OutputChannelInfo from "../model/OutputChannelInfo";
-import StatusIndicator from "../general/StatusIndicator";
-import DeviceName from "./DeviceName";
-import DeviceInfo from "../model/DeviceInfo";
+import ChannelDetailsTable from "../../devicelist/ChannelDetailsTable";
+import InputChannelInfo from "../../model/InputChannelInfo";
+import OutputChannelInfo from "../../model/OutputChannelInfo";
+import StatusIndicator from "../../general/StatusIndicator";
+import DeviceName from "../DeviceName";
+import DeviceInfo from "../../model/DeviceInfo";
 
-export default class DeviceDetailsConciseRow extends React.Component {
-  static getPropertyDisplayName(name) {
+export default class DeviceInfoRow extends React.Component {
+  constructor(props) {
+    super(props);
+    this.getPropertyDisplayName = this.getPropertyDisplayName.bind(this);
+    this.createInnerTable = this.createInnerTable.bind(this);
+    this.createTableCellContents = this.createTableCellContents.bind(this);
+  }
+
+  getPropertyDisplayName() {
```

Comment:
are we permanently removing additional info/extras from device info or are we keeping it for the future? i know we got rid of it for streams but i don't remember if we wanted to do the same for devices

Reply:
The "extras" aren't real data that we're handling anyways and we aren't adding a backend for it and have no plans to do so. because of that, I don't think it's worth adding a card for that to hold useless information. 
Since we will be adding Configuration info to the Device Info object shortly, that change can officially be made in that PR, however I don't see the need display it rn.